integration_test/go_java_launcher_integration_test.go

```diff
@@ -38,11 +38,35 @@ func TestMainMethod(t *testing.T) {
 	require.NoError(t, err, "failed: %s", output)
 
 	// part of expected output from launcher
-	assert.Regexp(t, `Argument list to executable binary: \[.+/bin/java -Xmx4M -Xmx1g -classpath .+/github.com/palantir/go-java-launcher/integration_test/testdata Main arg1\]`, output)
+	assert.Regexp(t, `Argument list to executable binary: \[.+/bin/java -Xmx4M -Xmx1g -classpath .+/integration_test/testdata Main arg1\]`, output)
```

Comment:
When running this test locally, the path is rendered as per where my machine is cloned, so I remove the unnecessary bits.